src/main/java/flight/reservation/flight/Flight.java

```diff
@@ -30,21 +31,12 @@ private void checkValidity() throws IllegalArgumentException {
 
     private boolean isAircraftValid(Airport airport) {
         return Arrays.stream(airport.getAllowedAircrafts()).anyMatch(x -> {
```

Comment:
```suggestion
        return Arrays.stream(airport.getAllowedAircrafts()).anyMatch(allowedAircraft -> {
```

Replace `x` with something more descriptive.